src/App.js

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
 import { Route, BrowserRouter as Router, Switch, Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
 import './App.scss';
 import About from './components/About';
 import Home from './components/Home';
 
 function App() {
+  const counter = useSelector((state) => state.counterReducer);
+  const isLogged = useSelector((state) => state.loggedReducer);
   return (
     <Router>
```

Comment:
We don't use common Router anymore, routing is handled by `ConnectedRouter`. Existing of Router in here prevents from pushing actions to redux.